federator/package.json

```diff
@@ -15,7 +15,9 @@
     "start": "node ./built/federator/src/main.js",
     "start-js": "node ./src/main.js",
     "build-start": "npm run build && npm run start",
-    "build": "npm run clean && tsc",
+    "build-start-dev": "npm run build-dev && npm run start",
```

Comment:
maybe add a clean call before build-dev?

Reply:
build-dev already does the clean

Reply:
![image](https://user-images.githubusercontent.com/17556614/133107076-4f92696a-6762-4516-974a-1243f5bf2e30.png)
We already have a clean call, or is not this what you mean?

Reply:
oh what a moron, didn't see that we're calling the build-dev from build-start-dev, just ignore the comment!